tests/Util/FormBuilderIteratorTest.php

```diff
@@ -16,27 +16,26 @@
 use PHPUnit\Framework\TestCase;
 use Sonata\AdminBundle\Util\FormBuilderIterator;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilder;
-use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormFactory;
+use Symfony\Component\Form\FormRegistryInterface;
 
 /**
  * @author Mike Meier <mike.meier@ibrows.ch>
  */
 final class FormBuilderIteratorTest extends TestCase
 {
-    private EventDispatcherInterface $dispatcher;
```

Comment:
I've removed `dispatcher` and `factory` properties from the class and replaced them with vars inside `setUp` since they are not used anywhere else.